src/main/kotlin/team/mozu/dsm/global/exception/s3/ImageNotFoundException.kt

```
@@ -0,0 +1,7 @@
+package team.mozu.dsm.global.exception.s3
+
+import team.mozu.dsm.global.error.exception.ErrorCode
+import team.mozu.dsm.global.error.exception.MozuException
+
+object ImageNotFoundException : MozuException(ErrorCode.IMAGE_NOT_FOUND) {
+}
```

detekt (1.23.8)

[warning] The class or object ImageNotFoundException is empty.

(detekt.empty-blocks.EmptyClassBlock)